src/views/notifications/index.tsx

```diff
@@ -27,6 +27,6 @@ const Notifications: FC = (): null => {
 		}
 	}, []);
 
-	return null;
+	return <></>;
```

Comment:
Why do we render a fragment?

Reply:
because I noticed with null sometimes notifications are not rendered on safari